src/sentry/migrations/0553_squash_same_name_repos.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 3.2.20 on 2023-09-15 00:00
+
+from django.db import migrations, router, transaction
+from django.db.models import Count, Q
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def squash_duped_repos(apps, schema_editor):
+    Repository = apps.get_model("sentry", "Repository")
+    PullRequest = apps.get_model("sentry", "PullRequest")
+    ReleaseHeadCommit = apps.get_model("sentry", "ReleaseHeadCommit")
+    LatestRepoReleaseEnvironment = apps.get_model("sentry", "LatestRepoReleaseEnvironment")
+    Commit = apps.get_model("sentry", "Commit")
+
+    duplicates = Repository.objects.values("name").annotate(count=Count("name")).filter(count__gt=1)
```

Comment:
@cathteng we probably need to use the ranged query set thing

Reply:
when you do `values("name")`, you just get a list of repo names that have duplicates, not a `BaseQueryset`

Reply:
@cathteng That's going to be generating 12k of them in memory. Might be Ok. Also, shouldn't you check if the org id is the same? It's ok for 2 names to be the same on different orgs.

Reply:
oh yeah

Reply:
oh jk this is still a BaseQuerySet so i can use RangeQuerySetWrapper